docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -0,0 +1,167 @@
+   +-------------+---------------+------------------+------------------+
+   | 14          | 2026-11-12    | 2026-05-12       | 2021-11-11       |
+   +-------------+---------------+------------------+------------------+
+
+.. _aiven-for-kafka:
+
+Aiven for Apache Kafka®
+-----------------------
+
+Starting with v2.5, Aiven for Kafka® ``major.minor`` version will reach
+EOL one year after it's made available on Aiven platform.
+
+.. container:: intercom-interblocks-table-container
+
+   =========== ============= ========================================
+   **Version** **Aiven EOL** **Availability start on Aiven Platform**
+   1.0.x       2021-02-01    2017-11-01
+   1.1.x       2021-02-01    2018-07-31
+   2.0.x       2021-02-01    2018-07-30
+   2.1.x       2021-02-01    2018-12-04
+   2.2.x       2021-02-01    2019-04-19
+   2.3.x       2021-08-13    2019-09-05
+   2.4.x       2021-08-13    2019-10-21
+   2.5.x       2021-08-13    2020-05-05
+   2.6.x       2021-08-13    2020-08-13
+   2.7.x       2022-01-24    2021-01-21
+   2.8.x       2022-06-02    2021-04-26
+   3.0         2022-11-22    2021-11-22
+   3.1         2023-02-14    2022-02-14
+   =========== ============= ========================================
+
+.. _h_0f2929c770:
+
+Aiven for Cassandra®
+--------------------
+
+Starting with v4, Aiven for Cassandra® ``major`` version will reach EOL
+six months after it's made available on Aiven platform.
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+------------------+------------------+
+   | **Version** | **Aiven EOL** | **Availability   | **Availability   |
+   |             |               | end on Aiven     | start on Aiven   |
+   |             |               | Platform**       | Platform**       |
+   +-------------+---------------+------------------+------------------+
+   | 3           | 2022-07-27    | 2022-04-27       | 2018-11-08       |
+   +-------------+---------------+------------------+------------------+
+   | 4           | N/A           | N/A              | 2021-12-09       |
+   +-------------+---------------+------------------+------------------+
+
+Aiven service EOL policy for major versions
+-------------------------------------------
+
+Aiven EOL policy is applicable only for services whose major versions
+are controlled by the customer.
+
+It applies to both **powered-on** and **powered-off** services running
+the affected versions.
+
+EOL notification
+~~~~~~~~~~~~~~~~
+
+When Aiven defines the EOL date for a service major version,
+
+-  Customers will receive an EOL email announcement along with
+   instructions on the next steps.
+
+-  Aiven Console will also show an EOL alert for affected services.
+
+-  Email reminders will be sent to customers on a monthly cadence. On
+   the month of the EOL date, the cadence shifts to weekly reminders.
+
+Upon EOL
+~~~~~~~~
+
+-  Affected powered-on services will be automatically upgraded to the
```

Comment:
Is this old stuff? We're not offering EL anymore

Reply:
removed EL section